.github/workflows/on-pages.yaml

```diff
@@ -0,0 +1,24 @@
+name: on-pages
+run-name: on pages deploy
+
+on:
+  workflow_run:
+    workflows: [pages-build-deployment]
+    types:
+      - completed
+    branches:
+      - gh-pages
+
+permissions:
+  contents: write
+
+jobs:
+  update-url:
+    # only run on user instance of template, not template itself
+    if: github.repository != 'vincerubinetti/lab-website-template'
```

Comment:
Should this test greenelab/lab-website-template? or both this and vincerubinetti if helpful for testing

Reply:
Yes it should definitely be greenelab. That was going to be the last thing I commit before merging. I'm keeping it for now because I'm still doing testing of the GitHub Actions on my personal development repo.